sos/plugins/logs.py

```diff
@@ -52,7 +52,8 @@ def setup(self):
                 self.add_copy_spec(i)
 
         if self.get_option('all_logs'):
-            self.add_journal(boot="this", allfields=True, output="verbose")
+            print("Getting journal --since %s" % (self.get_option("since")))
```

Comment:
Can't have plugin's calling `print`. Use `_log_info()` if you need to log anything, but I don't think this needs an explicit message.